src/libraries/System.Text.RegularExpressions/tests/Regex.GetGroupNames.Tests.cs

```diff
@@ -9,10 +9,11 @@ namespace System.Text.RegularExpressions.Tests
     public class GetGroupNamesTests
     {
         [Theory]
-        [InlineData("(?<first_name>\\S+)\\s(?<last_name>\\S+)", new string[] { "0", "first_name", "last_name" })]
-        public void GetGroupNames(string pattern, string[] expectedGroupNames)
+        [InlineData("(?<first_name>\\S+)\\s(?<last_name>\\S+)", RegexOptions.None, new string[] { "0", "first_name", "last_name" })]
```

Comment:
maybe Compiled also?

Reply:
I can add that here but in many other cases the code being tested is really independent of options (only dependent if the DFA option is used, in those cases I started using options as the last parameter whose default value is None unless provided otherwise, in which cases I don't have to change the existing data generators, just add extra  ones with last parameter being the DFA option at the end)